README.md

```diff
@@ -1,73 +1,44 @@
-![Google Cloud Platform Logo](https://cloud.google.com/_static/images/gcp-logo.png)
-# Google Cloud Tools for IntelliJ plugin
+# Cloud Tools for IntelliJ plugin
```

Comment:
Should we consider something like `Cloud Tools for the JetBrains family of IDEs` to emphasize our pretty broad support for other IDEs and languages?

Reply:
Hmm. I see your point, but we decide to stick with “IntelliJ” in the name (post-rename) so it wouldn’t make much sense to change it in the interim.